### PR TITLE
Create playbook-permissions.rst

### PR DESCRIPTION
**Pull request by justinegeffen**
Adding permissions content with roles. 

@stephenvanhemmen, @crspeller: this is the first draft but I'd love your feedback on whether the examples are what you're looking for. Any additional feedback or comments - particularly around the granularity and context - will be helpful. :)

Docs for: https://github.com/mattermost/mattermost-server/pull/19096

### CODE REVIEW

File: source/playbooks/playbook-permissions.rst [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+* **Team Administrators:** Permissions apply to all Team Admins, in all teams.
+
+For more information about System and Team Override Schemes, refer to the `Advanced Permissions <https://docs.mattermost.com/onboard/advanced-permissions.html>`__ documentation.
+
+Roles
+-----
+
+**Member**
+
+In the context of Playbooks, members are users of Mattermost who belong to the team within which the run is active and who may have a vested interest in the run. Depending on the permissions granted, members can have the same privileges as Playbook Admins or restricted privileges. Members are also referred to as participants in a run.
```
Comment (crspeller) [inline source/playbooks/playbook-permissions.rst:24]: Members are not the same as participants. Playbooks have members runs have participants.

File: source/playbooks/playbook-permissions.rst [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+**Playbook Admin**
+
+In the context of Playbooks, Playbook Admins are also members, but have elevated permissions to change playbook and run visibility and functional settings. The Playbook Admin role is a default role, and members need to be promoted to the role from within Playbooks. The Playbook Admin role then applies across the Playbooks platform. They do not have access to the System Console and their privileges are managed by System Admin.
```
Comment (crspeller) [inline source/playbooks/playbook-permissions.rst:28]: > The Playbook Admin role then applies across the Playbooks platform.

It's not. It's per playbook. 

> but have elevated permissions to change playbook and run visibility and functional settings

They may have elevated permissions depending on the scheme.

File: source/playbooks/playbook-permissions.rst [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+
+Manage playbook visibility
+--------------------------
+
+The default Playbooks settings are completely open. This means that members can add and remove other members, edit playbooks, convert Public playbooks to Private playbooks and so on. The default settings allow all members to participate in runs, edit playbooks, view runs and playbooks, remove other members from runs, edit actions and make other changes. Permissions provide better control over confidential runs and playbooks, as well as member management.
```
Comment (crspeller) [inline source/playbooks/playbook-permissions.rst:33]: The conversion one is not technically open by default. But since the manage membership is, you can always give yourself admin privileges.

File: source/playbooks/playbook-permissions.rst [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+Create read-only playbooks
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In this example, set permissions to only allow Playbook Admins to edit playbooks and start runs. Team members can view Public and Private playbooks and runs but can't edit playbooks, check off tasks, or remove members from runs.
+
+1. Go to **System Console > User Management > Permissions**.
+2. In the **All Members** section, uncheck **Manage Public Playbooks** and uncheck **Manage Private Playbooks**.
```
Comment (crspeller) [inline source/playbooks/playbook-permissions.rst:41]: This would also restrict members from creating public/private playbooks. Not sure that's the intent here. 
To avoid this you would have to unselect the individual permissions except the creation ones.

File: source/playbooks/playbook-permissions.rst [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+3. Scroll down to the **Playbook Admin** section and confirm that **Manage Public Playbooks** and **Manage Private Playbooks** are checked.
+4. Select **Save**.
+
+Manage how playbooks are accessed
+---------------------------------
+
+By default, playbooks are Public. This means that any member of the team can view the playbook and associated runs. Playbooks can be converted from Public to Private, but once a playbook is Private it can't be converted back to Public.
```
Comment (crspeller) [inline source/playbooks/playbook-permissions.rst:48]: I guess this is true because the create dialog does default to public. But the user is always prompted to select so there really isn't a default.

**Review by crspeller — CHANGES_REQUESTED** [review]

**Review by stephenvanhemmen — CHANGES_REQUESTED** [review]

File: source/playbooks/playbook-permissions.rst (hunk 3 above)
Comment (justinegeffen) [inline source/playbooks/playbook-permissions.rst:33]:
```suggestion
The default Playbooks settings are completely open and the default settings allow all members to participate in runs, edit playbooks, view runs and playbooks, remove other members from runs, edit actions and make other changes. Permissions provide better control over confidential runs and playbooks, as well as member management.
```

File: source/playbooks/playbook-permissions.rst [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+Restrict who can convert playbooks from Public to Private
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In this example, you can control whether Members can convert playbooks from Public to Private.
+
+1. Go to **System Console > User Management > Permissions**.
+2. In the **All Members** section, check **Convert Playbooks**.
+3. Select **Save**.
+
+To restrict this action to only Members and not allow Playbook Admins to convert playbooks from Public to Private:
```
Comment (justinegeffen) [inline source/playbooks/playbook-permissions.rst:59]:
```suggestion
To restrict this action so only Playbook Admins can convert playbooks from Public to Private:
```

File: source/playbooks/playbook-permissions.rst (hunk 1 above)
Comment (justinegeffen) [inline source/playbooks/playbook-permissions.rst:24]:
```suggestion
In the context of Playbooks, members are users of Mattermost who belong to a playbook or are a Playbook Admin.
```

File: source/playbooks/playbook-permissions.rst [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+Playbooks roles
+---------------
+
+**Member**
+
+In the context of Playbooks, members are users of Mattermost who belong to a playbook or are a Playbook Admin.
+
+**Participant**
+
+In the context of Playbooks, participants belong to or take part in a run. They can be a member or a Playbook Admin.
```
Comment (justinegeffen) [inline source/playbooks/playbook-permissions.rst:28]: @crspeller, I'm hitting a blank: Do participants have to be part of a playbook or can they participate in a run without being a member of a playbook?

File: source/playbooks/playbook-permissions.rst [hunk 8]
```diff
@@ -0,0 +1,78 @@
+Roles and Permissions
+=====================
+
+There are different ways to access and interact with playbooks and this is controlled in the System Console in the form of permissions. Permissions can be granted in a variety of ways, to allow for different combinations of access and visibility, and to different roles. Two permission schemes are provided in Mattermost:
```
Comment (cwarnermm) [inline source/playbooks/playbook-permissions.rst:4]:
```suggestion
There are different ways to access and interact with playbooks, and access is controlled in the System Console in the form of permissions. Permissions can be granted in a variety of ways, to allow for different combinations of access, visibility, and roles. Two permission schemes are provided in Mattermost:
```

File: source/playbooks/playbook-permissions.rst [hunk 9]
```diff
@@ -0,0 +1,78 @@
...
+
+In the context of Playbooks, members are users of Mattermost who belong to a playbook or are a Playbook Admin.
+
+**Participant**
+
+In the context of Playbooks, participants belong to or take part in a run. They can be a member or a Playbook Admin.
+
+**Playbook Admin**
+
+In the context of Playbooks, Playbook Admins are also members, and may have elevated permissions to change playbook and run visibility and functional settings. The Playbook Admin role is a default role, and members need to be promoted to the role from within Playbooks. The Playbook Admin role is applied per playbook. They do not have access to the System Console and their privileges are managed by System Admin.
```
Comment (cwarnermm) [inline source/playbooks/playbook-permissions.rst:32]:
```suggestion
In the context of Playbooks, Playbook Admins are also members, and may have elevated permissions to change playbook and run visibility as well as functional settings. The Playbook Admin role is a default role, and members need to be promoted to the role from within Playbooks. The Playbook Admin role is applied per playbook. They do not have access to the System Console and their privileges are managed by the System Admin.
```

File: source/playbooks/playbook-permissions.rst [hunk 10]
```diff
@@ -0,0 +1,78 @@
...
+
+Manage playbook visibility
+--------------------------
+
+The default Playbooks settings are completely open and the default settings allow all members to participate in runs, edit playbooks, view runs and playbooks, remove other members from runs, edit actions and make other changes. Permissions provide better control over confidential runs and playbooks, as well as member management.
```
Comment (cwarnermm) [inline source/playbooks/playbook-permissions.rst:37]:
```suggestion
The default Playbooks settings are completely open which enable all members to participate in runs, edit playbooks, view runs and playbooks, remove other members from runs, edit actions, and make other changes. Permissions provide better control over confidential runs and playbooks, as well as member management.
```

File: source/playbooks/playbook-permissions.rst [hunk 11]
```diff
@@ -0,0 +1,78 @@
...
+
+Create read-only playbooks
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In this example, set permissions to only allow Playbook Admins to edit playbooks and start runs. Team members can view Public and Private playbooks and runs but can't edit playbooks, check off tasks, or remove members from runs.
```
Comment (cwarnermm) [inline source/playbooks/playbook-permissions.rst:42]:
```suggestion
In the following example, Playbook Admins can edit playbooks and start runs. Team members can view Public and Private playbooks and runs, but can't edit playbooks, check off tasks, or remove members from runs.
```

File: source/playbooks/playbook-permissions.rst [hunk 12]
```diff
@@ -0,0 +1,78 @@
...
+
+1. Go to **System Console > User Management > Permissions**.
+2. In the **All Members** section, uncheck **Manage Public Playbooks** and uncheck **Manage Private Playbooks**.
+3. Scroll down to the **Playbook Admin** section and confirm that **Manage Public Playbooks** and **Manage Private Playbooks** are checked.
```
Comment (cwarnermm) [inline source/playbooks/playbook-permissions.rst:46]: "checked" or "selected"?

File: source/playbooks/playbook-permissions.rst [hunk 13]
```diff
@@ -0,0 +1,78 @@
...
+
+Manage how playbooks are accessed
+---------------------------------
+
+By default, playbooks are Public. This means that any member of the team can view the playbook and associated runs. Playbooks can be converted from Public to Private, but once a playbook is Private it can't be converted back to Public.
+
+Restrict who can convert playbooks from Public to Private
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In this example, you can control whether Members can convert playbooks from Public to Private.
```
Comment (cwarnermm) [inline source/playbooks/playbook-permissions.rst:57]:
```suggestion
You can control whether Members can convert playbooks from Public to Private.
```

File: source/playbooks/playbook-permissions.rst [hunk 14]
```diff
@@ -0,0 +1,78 @@
...
+
+1. Go to **System Console > User Management > Permissions**.
+2. In the **All Members** section, check **Convert Playbooks**.
```
Comment (cwarnermm) [inline source/playbooks/playbook-permissions.rst:60]: "check" or "select"? Now may be a good time to align on permissions terminology. Are users "enabling/disabling" permissions, or are they selecting/checking individual permission options?

**Review by cwarnermm — COMMENTED** [review]
Feedback provided inline.

File: source/playbooks/playbook-permissions.rst (hunk 7 above)
Comment (crspeller) [reply source/playbooks/playbook-permissions.rst:28]: The participants list in the RHS is all the members of the run's channel excluding the owner and any bots.

File: source/playbooks/playbook-permissions.rst (hunk 14 above)
Comment (justinegeffen) [reply source/playbooks/playbook-permissions.rst:60]: We use "check", "uncheck", and "enable" in the current docs. I can't find mention of "disable". I propose aligning with that. :)

File: source/playbooks/playbook-permissions.rst [hunk 15]
```diff
@@ -0,0 +1,78 @@
...
+
+Manage playbook visibility
+--------------------------
+
+The default Playbooks settings are completely open and the default settings allow all members to participate in runs, edit playbooks, view runs and playbooks, remove other members from runs, edit actions and make other changes. Permissions provide better control over confidential runs and playbooks, as well as member management.
+
+Create read-only playbooks
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In the following example, Playbook Admins can edit playbooks and start runs. Team members can view Public and Private playbooks and runs, but can't edit playbooks, check off tasks, or remove members from runs.
```
Comment (justinegeffen) [inline source/playbooks/playbook-permissions.rst:42]:
```suggestion
In the following example, only Playbook Admins can edit playbooks. Team members can view Public and Private playbooks and runs, but can't edit playbooks, check off tasks, or remove members from runs.
```

File: source/playbooks/playbook-permissions.rst [hunk 16]
```diff
@@ -0,0 +1,78 @@
...
+
+1. Go to **System Console > User Management > Permissions**.
+2. In the **All Members** section, uncheck **Manage Public Playbooks** and uncheck **Manage Private Playbooks**.
+3. Scroll down to the **Playbook Admin** section and confirm that **Manage Public Playbooks** and **Manage Private Playbooks** are checked.
+4. Select **Save**.
+
+Manage how playbooks are accessed
+---------------------------------
+
+By default, playbooks are Public. This means that any member of the team can view the playbook and associated runs. Playbooks can be converted from Public to Private, but once a playbook is Private it can't be converted back to Public.
```
Comment (justinegeffen) [inline source/playbooks/playbook-permissions.rst:52]:
```suggestion
Playbooks are Public. This means that any member of the team can view the playbook and associated runs. Playbooks can be converted from Public to Private, but once a playbook is Private it can't be converted back to Public.
```

File: source/playbooks/playbook-permissions.rst (hunk 12 above)
Comment (justinegeffen) [reply source/playbooks/playbook-permissions.rst:46]: Checked. :)

File: source/playbooks/playbook-permissions.rst [hunk 17]
```diff
@@ -0,0 +1,89 @@
...
+* **All Members:** Permissions apply to all members, including Admins, in all channels, in all teams.
+* **Channel Administrators:** Permissions apply to all Channel Admins in all channels, in all teams.
+* **Playbook Admins:** Permissions apply to all Playbook Admins in all channels, in all teams.
+* **Team Administrators:** Permissions apply to all Team Admins, in all teams.
+
+For more information about System and Team Override Schemes, refer to the `Advanced Permissions <https://docs.mattermost.com/onboard/advanced-permissions.html>`__ documentation.
+
+.. note::
+
+  Before changing permissions, ensure that you or another member have Admin access in Playbooks otherwise changing configurations could result in being unable to use Playbooks.
```
Comment (stephenvanhemmen) [inline source/playbooks/playbook-permissions.rst:21]: Might be helpful to show how they can do this: navigate to a playbook -> `Manage Access` menu -> change their role from member to playbook admin

File: source/playbooks/playbook-permissions.rst [hunk 18]
```diff
@@ -0,0 +1,89 @@
...
+
+1. Go to **System Console > User Management > Permissions**.
+2. In the **All Members** section, check **Convert Playbooks**.
+3. Select **Save**.
+
+To restrict this action so only Playbook Admins can convert playbooks from Public to Private:
+
+1. Go to **System Console > User Management > Permissions**.
+2. Scroll down to **Playbook Admin**.
+3. Uncheck **Convert Playbooks**.
```
Comment (stephenvanhemmen) [inline source/playbooks/playbook-permissions.rst:78]: Wouldn't this prevent Playbook Admins from converting playbook, instead of allowing them to do so?

**Review by cwarnermm — APPROVED** [review]
Thanks!

File: source/playbooks/playbook-permissions.rst (hunk 17 above)
Comment (justinegeffen) [reply source/playbooks/playbook-permissions.rst:21]: Great point, thank you!

File: source/playbooks/playbook-permissions.rst (hunk 18 above)
Comment (justinegeffen) [reply source/playbooks/playbook-permissions.rst:78]: Yep you're right. Will rectify.

**Review by stephenvanhemmen — APPROVED** [review]
The role definitions and permissions recommendations are great. Thanks! 🎉

**Comment by justinegeffen** [comment]
@crspeller, this is ready for your review. :)

**Review by lieut-data — CHANGES_REQUESTED** [review]
A few thoughts below, but TLDR; I'm having trouble internalizing the way this feature works.

I feel like we need to take a step back and work exclusively from the use cases that advanced permissions solve, then describe each within the specifics of the feature, and finally call out the step-by-step instructions to enable same.

For example, a common ask for community is to restrict who can run a given playbook. I think our options here are to either make such playbooks private (the current solution) OR to disable running any playbook unless you're a playbook administrator, which in turn means that members can never edit any playbook (otherwise they can make themselves admins on any playbook).

Thoughts on next steps?

File: source/playbooks/playbook-permissions.rst (hunk 7 above)
Comment (justinegeffen) [reply source/playbooks/playbook-permissions.rst:34]: Good point. I was trying to work the definitions into the documentation and this was a late addition. Happy to remove.

File: source/playbooks/playbook-permissions.rst [hunk 19]
```diff
@@ -0,0 +1,91 @@
...
+* **Guests:** If Guest Accounts are enabled, permissions apply to guest users in all channels, in all teams.
+* **All Members:** Permissions apply to all members, including Admins, in all channels, in all teams.
+* **Channel Administrators:** Permissions apply to all Channel Admins in all channels, in all teams.
+* **Playbook Admins:** Permissions apply to all Playbook Admins in all channels, in all teams.
+* **Team Administrators:** Permissions apply to all Team Admins, in all teams.
+
+For more information about System and Team Override Schemes, refer to the `Advanced Permissions <https://docs.mattermost.com/onboard/advanced-permissions.html>`__ documentation.
+
+In the context of Playbooks, members are assigned a role and based on the selected permissions, this determines how they interact with Playbooks. A member can be a member of one playbook, and an admin of another. This allows for granular permissions across teams and departments.
+
```
[the remark this answers is not in the document]
Comment (justinegeffen) [reply source/playbooks/playbook-permissions.rst:20]: I like the idea of describing the permission scheme. Would you describe the default permission scheme as "open to all" or something more elaborate like "The default permission scheme allows all members of a team to access playbooks, start runs, and edit configurations such as actions."

File: source/playbooks/playbook-permissions.rst [hunk 20]
```diff
@@ -0,0 +1,91 @@
...
+.. note::
+
+Permissions are applied across Playbooks as a whole, so it's important to set your roles up first before editing permissions. To ensure that you don't lose access to playbooks, navigate to the playbook you're a member of. Select the **Manage Access** icon and change your role from **Member** to **Admin** 
+
+Playbook roles
+---------------
+
+**Member**
+
+In the context of Playbooks, members are users of Mattermost who belong to a playbook or are a Playbook Admin.
```
Comment (justinegeffen) [inline source/playbooks/playbook-permissions.rst:30]:
```suggestion
In the context of Playbooks, members are users of Mattermost who are added to a playbook.
```

File: source/playbooks/playbook-permissions.rst [hunk 21]
```diff
@@ -0,0 +1,87 @@
...
+
+1. Go to **System Console > User Management > Permissions**.
+2. In the **All Members** section, check **Convert Playbooks**.
+3. Select **Save**.
+
+To restrict this action so only Playbook Admins can convert playbooks from public to private:
+
+1. Go to **System Console > User Management > Permissions**.
+2. Scroll down to **Playbook Admin**.
+3. Uncheck **Convert Playbooks**.
```
Comment (justinegeffen) [inline source/playbooks/playbook-permissions.rst:76]:
```suggestion
3. Check **Convert Playbooks**.
```

File: source/playbooks/playbook-permissions.rst [hunk 22]
```diff
@@ -0,0 +1,87 @@
...
+3. Select **Save**.
+
+To restrict this action so only Playbook Admins can convert playbooks from public to private:
+
+1. Go to **System Console > User Management > Permissions**.
+2. Scroll down to **Playbook Admin**.
+3. Check **Convert Playbooks**.
+4. Select **Save**.
+
+I want to control who starts a run
```
Comment (justinegeffen) [inline source/playbooks/playbook-permissions.rst:79]: @lieut-data, you mentioned: "For example, a common ask for community is to restrict who can run a given playbook. I think our options here are to either make such playbooks private (the current solution) OR to disable running any playbook unless you're a playbook administrator, which in turn means that members can never edit any playbook (otherwise they can make themselves admins on any playbook)."

Does the example below address this? The way I understand it, it does however it only deals with running playbooks - not editing them. From what I can see, you can separate starting a run from editing playbooks. But the permissions can be very finely curated to allow members to create a playbook, not allow them to create checklists, actions, and templates, but still allow them to start a run.

I do see a benefit in describing that level of granularity, especially if it's been asked for.

File: source/playbooks/playbook-permissions.rst [hunk 23]
```diff
@@ -0,0 +1,85 @@
...
+There are different ways for teams to access and interact with playbooks. This is managed in the System Console using permissions. Permissions can be granted in a variety of ways, to allow for different combinations of access and visibility.
+
+Permissions are provided using:
+
+* **System Scheme:** Applies permissions universally across all teams, channels, and playbooks.
+* **Team Override Schemes:** Allow admins to customize permissions for each team (available in Mattermost Enterprise and Professional).
+
+For more information about System and Team Override Schemes, refer to the `Advanced Permissions <https://docs.mattermost.com/onboard/advanced-permissions.html>`__ documentation.
+
+In the context of Playbooks, members are assigned a role and based on the selected permissions, this determines how they interact with Playbooks. A member can be a member of one playbook, and an admin of another. This allows for granular permissions across teams and departments.
```
Comment (justinegeffen) [inline source/playbooks/playbook-permissions.rst:13]:
```suggestion
In the context of Playbooks, members are assigned a role and based on the selected permissions, this determines how they interact with Playbooks. A member can be a member of one playbook, and an admin of another. This allows for granular permissions across teams and departments. For example, setting playbook visibility so only certain teams can view it, or setting permissions to allow an organization to view a playbook but only designated team members can make edits.
```

**Comment by justinegeffen** [comment]
@lieut-data, I've added some minor updates. Let me know if they're moving in the right direction. :)

File: source/playbooks/playbook-permissions.rst [hunk 24]
```diff
@@ -0,0 +1,85 @@
...
+
+In the context of Playbooks, members are users of Mattermost who are added to a playbook.
+
+**Playbook Admin**
+
+In the context of Playbooks, Playbook Admins are also members, and may have elevated permissions to change playbook and run visibility as well as functional settings. They do not have access to the System Console and their privileges are managed by the System Admin. Members need to be promoted to the role from within Playbooks. The Playbook Admin role is applied per playbook.
+
+.. note::
+
+Before you make changes to permissions and to ensure that you don't lose access to playbooks, . To do this, navigate to the playbook you're a member of. Select the **Manage Access** icon and change your role from **Member** to **Admin**.
```
Comment (lieut-data) [inline source/playbooks/playbook-permissions.rst:30]:
```suggestion
Before you make system or team changes to permissions, ensure that you don't lose access to your existing playbooks. Navigate to the playbook you're a member of. Select the **Manage Access** icon and change your role from **Member** to **Admin**.
```

File: source/playbooks/playbook-permissions.rst [hunk 25]
```diff
@@ -0,0 +1,85 @@
...
+
+Playbooks permissions
+---------------------
+
+The default Playbooks settings are completely open which enable all members to participate in runs, edit playbooks, view runs and playbooks, remove other members from runs, edit actions, and make other changes. Permissions provide better control over confidential runs and playbooks, as well as member management.
```
Comment (lieut-data) [inline source/playbooks/playbook-permissions.rst:35]:
```suggestion
The default Playbooks settings are completely open which enable all members to participate in runs, edit playbooks, view runs and playbooks, remove other members from runs, edit actions, and make other changes. Permissions provide better control over confidential runs and playbooks, as well as member management. Note that even with the default settings, private playbooks restrict these actions to members of the playbook.
```

File: source/playbooks/playbook-permissions.rst [hunk 26]
```diff
@@ -0,0 +1,85 @@
...
+
+Create read-only playbooks
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In the following example, only Playbook Admins can edit playbooks. Team members can view public and private playbooks and runs, but can't edit playbooks, check off tasks, or remove members from runs. They also can't create public or private playbooks.
```
Comment (lieut-data) [inline source/playbooks/playbook-permissions.rst:40]: `check off tasks` is a run permission, not a playbook permission:
```sugestion
In the following example, only Playbook Admins can edit playbooks. Other users can view public playbooks and private playbooks of which they are a member, but can't edit any playbook or change playbook memberships.
```

File: source/playbooks/playbook-permissions.rst [hunk 27]
```diff
@@ -0,0 +1,85 @@
...
+
+1. Go to **System Console > User Management > Permissions**.
+2. In the **All Members** section, uncheck **Manage Public Playbooks** and uncheck **Manage Private Playbooks**.
+3. Scroll down to the **Playbook Admin** section and confirm that **Manage Public Playbooks** and **Manage Private Playbooks** are checked.
+4. Select **Save**.
+
+You can also set permissions for read-only playbooks that do allow members to create new public or private playbooks.
```
Comment (lieut-data) [inline source/playbooks/playbook-permissions.rst:47]: This section doesn't seem to "fit" in the "read-only" category. Should we make it a new section entitled, "Restricting who can create playbooks`?
```suggestion
Restricting who can create playbooks
-------------------------------------
You can also set permissions for read-only playbooks that do allow members to create new public or private playbooks.
```

File: source/playbooks/playbook-permissions.rst [hunk 28]
```diff
@@ -0,0 +1,85 @@
...
+1. Go to **System Console > User Management > Permissions**.
+2. In the **All Members** section, uncheck **Manage Public Playbooks** and uncheck **Manage Private Playbooks**.
+3. Then, check **Create Public Playbook** and **Create Private Playbook**.
+4. Select **Save**.
+
+I want to manage how playbooks are accessed
+-------------------------------------------
+
+A public playbook and its associated runs can be viewed by any member of a team. Playbooks can be converted from public to private, but once a playbook is private it can't be converted back to public.
+
```
Comment (lieut-data) [inline source/playbooks/playbook-permissions.rst:58]: This section doesn't seem to "fit" -- it has no actions, for example. Maybe just remove?
```suggestion
```

File: source/playbooks/playbook-permissions.rst [hunk 29]
```diff
@@ -0,0 +1,85 @@
...
+I want to restrict who can convert playbooks from public to private
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can control whether Members can convert playbooks from public to private.
+
+1. Go to **System Console > User Management > Permissions**.
+2. In the **All Members** section, check **Convert Playbooks**.
+3. Select **Save**.
+
+To restrict this action so only Playbook Admins can convert playbooks from public to private:
```
Comment (lieut-data) [inline source/playbooks/playbook-permissions.rst:68]: This reads like it's a necessary next step for admins, but if they do the above, admins should inherit this by default. Maybe rephrase as,
```suggestion
Alternatively, to restrict this action so only Playbook Admins can convert playbooks from public to private, uncheck the setting above and:
```

File: source/playbooks/playbook-permissions.rst [hunk 30]
```diff
@@ -0,0 +1,85 @@
...
+4. Select **Save**.
+
+I want to control who starts a run
+----------------------------------
+
+By default, all Members can start a run using a playbook. You can restrict this so that only Playbook Admins can start a run. Note that with this configuration, Members are not able to start runs or edit playbooks.
+
+1. Go to **System Console > User Management > Permissions**.
+2. In the **All Members** section, uncheck **Manage Runs**. This also unchecks **Create Runs**.
+3. Scroll down to **Playbook Admin** and ensure that **Manage Runs** and **Create Runs** are checked.
```
Comment (lieut-data) [inline source/playbooks/playbook-permissions.rst:82]: @crspeller, is the `Manage Runs` setting necessary here to limit who can run a playbook? Wondering if we can make the example simpler by focussing on just the one knob.

**Review by lieut-data — APPROVED** [review]
Thanks, @justinegeffen! A few comments below, but letting you drive :)

File: source/playbooks/playbook-permissions.rst [hunk 31]
```diff
@@ -0,0 +1,82 @@
...
+
+Playbooks permissions
+---------------------
+
+The default Playbooks settings are completely open which enable all members to participate in runs, edit playbooks, view runs and playbooks, remove other members from runs, edit actions, and make other changes. Permissions provide better control over confidential runs and playbooks, as well as member management. Note that even with the default settings, private playbooks restrict these actions to members of the playbook.
+
+Create read-only playbooks
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+In the following example, only Playbook Admins can edit playbooks. Team members can view public and private playbooks and runs, but can't edit playbooks, check off tasks, or remove members from runs. They also can't create public or private playbooks.
```
Comment (justinegeffen) [inline source/playbooks/playbook-permissions.rst:40]:
```suggestion
In the following example, only Playbook Admins can edit playbooks. Other users can view public playbooks and private playbooks of which they are a member, but can't edit any playbook or change playbook memberships.
.
```

File: source/playbooks/playbook-permissions.rst (hunk 30 above)
Comment (crspeller) [reply source/playbooks/playbook-permissions.rst:80]: They are nested and `Create Runs` is the only child so they are basically the same box. Clicking on one with check/uncheck the other.

**Review by crspeller — APPROVED** [review]

File: source/playbooks/playbook-permissions.rst [hunk 32]
```diff
@@ -0,0 +1,83 @@
...
+4. Select **Save**.
+
+I want to control who starts a run
+----------------------------------
+
+By default, all Members can start a run using a playbook. You can restrict this so that only Playbook Admins can start a run. Note that with this configuration, Members are not able to start runs or edit playbooks.
+
+1. Go to **System Console > User Management > Permissions**.
+2. In the **All Members** section, uncheck **Manage Runs**. This also unchecks **Create Runs**.
+3. Scroll down to **Playbook Admin** and ensure that **Manage Runs** and **Create Runs** are checked.
```
Comment (justinegeffen) [inline source/playbooks/playbook-permissions.rst:80]:
```suggestion
3. Scroll down to **Playbook Admin** and ensure that **Manage Runs** is checked.
```
